uber/templates/accounts/update_password_of_other.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base-admin.html" %}
+{% block title %}Change Password {% endblock %}
+{% block content %}
+
+<h2 class="center"> Update Password for {{account.attendee.full_name}}</h2>
+
+<form method="post" action="update_password_of_other">
+{% csrf_token %}
+<input type="hidden" name="id" value="{{account.id}}"/ />
+
+<table>
+<tr class="center">
+    <td width="33%">
+        <b>Your Password:</b> <br/>
+        <input class="focus" type="password" name="updater_password" />
+    </td>
+    <td width="33%">
+        <b>New Password:</b> <br/>
+        <input type="password" id='new1' name="new_password" />
+    </td>
+    <td width="33%">
+        <b>Confirm New Password:</b> <br/>
+        <input type="password" id='new2' name="confirm_new_password" />
+    </td>
+</tr>
+</table>
+
+<div class="center">
+    <input type="submit" class="submit-button" value="Change Password" />
+</div>
+
+</form>
+
+    <script>
+    $(function() {
+    $(document.body).on('click', '.submit-button', function (event) {
+        event.preventDefault();
+            $(event.target).parents('form').submit();
+
+        else{
+            alert("New and confirmed passwords do not match.");
+        }
+    });
+});
+    </script>
```

Comment:
Also, since we're not relying on a client-side check here ,you can remove this entire ``<script>`` block as well.